eng/packaging.targets

```diff
@@ -219,15 +220,20 @@
       <None Include="$(_NETStandardCompatErrorFilePath)"
             PackagePath="buildTransitive\%(NETStandardCompatError.Identity)"
             Pack="true" />
+      <!-- Add the placeholder file for the supported target framework group if that directory is empty. -->
```

Comment:
If I wanted to fix [https://github.com/dotnet/runtime/issues/60870](https://github.com/dotnet/runtime/issues/60870), I would need `IncludeMultiTargetRoslynComponentTargetsInPackage` to run before this target, correct?

So I would need to switch around the PackDependsOn from

```xml
<PackDependsOn>AddNETStandardCompatErrorFileForPackaging;IncludeAnalyzersInPackage;$(PackDependsOn)</PackDependsOn>
```

to

```xml
<PackDependsOn>IncludeAnalyzersInPackage;AddNETStandardCompatErrorFileForPackaging;$(PackDependsOn)</PackDependsOn>
```

?

Reply:
Correct.